msbuild/Xamarin.MacDev.Tasks/MsBuildTasks/Delete.cs

```diff
@@ -25,5 +25,9 @@ public override bool Execute ()
 
 			return result;
 		}
+
+		public bool ShouldCopyToBuildServer (ITaskItem item) => false;
+
+		public bool ShouldCreateOutputFile (ITaskItem item) => false;
```

Comment:
@emaf is this all was I needed?

Reply:
I think you also need to implement `GetAdditionalItemsToBeCopied`, you can do something like this
``` public IEnumerable<ITaskItem> GetAdditionalItemsToBeCopied () => Enumerable.Empty<ITaskItem> ();```